lib/Cake/TestSuite/Fixture/CakeFixtureManager.php

```diff
@@ -303,4 +329,8 @@ public function shutDown() {
 		}
 	}
 
+	public function __destruct()
+	{
+		$this->shutDown();
```

Comment:
This is a bit gross but it's the only way I could find to call the shutDown() method at the end of the test run.